client/src/modules/Registry/components/ContactInfo/index.test.tsx

```diff
@@ -0,0 +1,133 @@
+import { cleanup, fireEvent, render, screen } from '@testing-library/react';
+import { Form } from 'antd';
+import { ERROR_MESSAGES, LABELS, PLACEHOLDERS, RSSCHOOL_BOT_LINK } from 'modules/Registry/constants';
+import { ContactInfo } from './index';
+
+const mockValues = {
+  contactsTelegram: 'telegram',
+  contactsSkype: 'skype',
+  contactsWhatsApp: 'whatsapp',
+  contactsEmail: 'test@test.com',
+  contactsPhone: '+123456789',
+  contactsNotes: 'notes',
+};
+
+const setup = () =>
+  render(
+    <Form initialValues={mockValues}>
+      <ContactInfo />
+    </Form>,
+  );
+
+describe('ContactInfo', () => {
+  afterEach(() => {
+    cleanup();
+  });
+
+  test('should render form items with proper values', async () => {
```

Comment:
Ideally should be several tests. Use `test.each` https://jestjs.io/docs/api#testeachtablename-fn-timeout
You can find examples in repository

Reply:
fixed